shared/constants/settings.js

```diff
@@ -111,7 +111,7 @@ export type PassphraseState = {
   newPassphrase: HiddenString,
   newPassphraseConfirm: HiddenString,
   showTyping: boolean,
-  errorMessage: ?HiddenString,
```

Comment:
I'm not sure if this error can't leak information, so I don't think we want to do this